core-module/lending.md

```diff
@@ -39,16 +39,34 @@ It was preferred to get inspiration from Yearn rather than using Yearn directly
 
 ## 💹 Debt Ratio
 
-For each strategy associated to a collateral, it is possible to compute a debt ratio that corresponds to the ratio between what has been lent and the total amount of collateral in the pool \(what's kept in reserves + what has been lent across all strategies\). Each strategy has its target debt ratio. Below this debt ratio, keepers can call the `harvest` function to give more collateral to the strategy and above this debt ratio, collateral can be withdrawn from the strategy and hence from the corresponding lending platforms.
+For each strategy associated to a collateral, it is possible to compute a debt ratio that corresponds to the ratio between what has been lent and the total amount of collateral in the pool (what's kept in reserves + what has been lent across all strategies). Each strategy has its target debt ratio. Below this debt ratio, keepers can call the `harvest` function to give more collateral to the strategy and above this debt ratio, collateral can be withdrawn from the strategy and hence from the corresponding lending platforms.
 
 {% hint style="info" %}
 Keepers cannot choose the amount they lend or withdraw from lending platforms: this is automatically computed using the strategy's target debt ratio at each call to `harvest.`
 {% endhint %}
 
-## ✖️ Back To The Multiplier Effect For SLPs
+## ✖️ Multiplier Effect For SLPs
 
 Thanks to the lending strategies, SLPs get rewards not only from their capital, but also from that of Users and HAs. This allow them to get potentially higher yield than if they were farming only with their capital. More info in the SLP page below.
 
 {% content-ref url="standard-liquidity-providers/README.md" %}
 [standard-liquidity-providers/README.md](standard-liquidity-providers/README.md#%E2%9C%96-multiplier-effect)
 {% endcontent-ref %}
+
+## 🤓 Strategies details
+
+### Base strategy
+
+The first strategies used in the Angle Core module have been forked from Yearn. They compare Aave and Compound markets, and deposit funds in both protocols in order to maximize the global yield. 
```

Comment:
Here we should rewrite the first paragraph where we explain that we lend to Compound and Aave: to make it more generic now. We should say that we have yield strategies, keep the explanation on how it works and then add details on the different strategies